libraries/AP_PiccoloCAN/AP_PiccoloCAN.cpp

```diff
@@ -670,7 +678,7 @@ bool AP_PiccoloCAN::is_esc_channel_active(uint8_t chan)
     // Check if a motor function is assigned for this motor channel
     SRV_Channel::Aux_servo_function_t motor_function = SRV_Channels::get_motor_function(chan);
 
-    if (SRV_Channels::function_assigned(motor_function)) {
+    if (SRV_Channels::function_assigned(motor_function) || SRV_Channels::channel_function(chan) == SRV_Channel::k_throttle) {
```

Comment:
Should have `k_throttleLeft` and `k_throttleRight` here too.

Reply:
This has been run up on an aircraft with a Currawong velocity ESC, but not yet flown. I'll make changes to include the left and right throttle functions as well